RetailApp/RetailApp.PaymentService/MapperConfig/PaymentGrpcProfile.cs

```diff
@@ -0,0 +1,18 @@
+﻿using AutoMapper;
+using RetailApp.BAL.Models;
+using RetailApp.PaymentService.Protos;
+
+namespace RetailApp.PaymentService.MapperConfig
+{
+    public class PaymentGrpcProfile : Profile
```

Comment:
Just PaymentProfile, no need again to specify service type. It's already in different namespace. Same goes for other naming for other profiles.